src/test/lib/operators/import_test.cpp

```diff
@@ -204,6 +207,31 @@ TEST_F(OperatorsImportTest, ReplaceExistingTable) {
   EXPECT_TABLE_EQ_ORDERED(Hyrise::get().storage_manager.get_table("a"), expected_table);
 }
 
+TEST_F(OperatorsImportTest, RetrieveCsvMetaFromEmptyTable) {
+  auto existing_table = CsvParser::create_table_from_meta_file("resources/test_data/csv/float.csv.json");
+
+  Hyrise::get().storage_manager.add_table("a", existing_table);
+
+  auto expected_table = load_table("resources/test_data/tbl/float.tbl");
+  auto importer = std::make_shared<Import>("resources/test_data/csv/float_without_json_meta.csv", "a");
+  importer->execute();
+
+  EXPECT_TABLE_EQ_ORDERED(Hyrise::get().storage_manager.get_table("a"), expected_table);
+}
+
+TEST_F(OperatorsImportTest, RetrieveCsvMetaFromTable) {
```

Comment:
Can you add a test where we append to an existing table? Or would that be out of scope for that PR?

Reply:
I didn't think that we should be able to append to an existing table. And I don't think that the current code can do that.